### PR TITLE
Adds the full path to the launch testing stdin

### PR DESCRIPTION
**Pull request by Voldivh**
This PR addresses an item from https://github.com/audrow/yatm/issues/366 according to this [discussion](https://github.com/osrf/ros2_test_cases/issues/737#issuecomment-1540107603).

### CODE REVIEW

File: requirements/features-executable-from-links.yaml [hunk 1]
```diff
@@ -27,22 +27,22 @@ requirements:
     checks:
       - name: Test `hello_world_launch_test.py`
         try:
-          - stdin: launch_test test/launch_testing/examples/hello_world_launch_test.py
+          - stdin: launch_test /opt/ros/iron/share/launch_testing/examples/hello_world_launch_test.py
```
Comment (clalancette) [inline requirements/features-executable-from-links.yaml:30]: The problem with this is that this is only correct when you are running the tests from Debians.  It is not correct for the binaries, or when building from source, or when running on Windows.  I'm honestly not sure what to do here; it is a tricky one to describe across all of the platforms.

**Review by clalancette — CHANGES_REQUESTED** [review]

File: requirements/features-executable-from-links.yaml (hunk 1 above)
Comment (Voldivh) [reply requirements/features-executable-from-links.yaml:30]: I did notice exactly that. However, I thought this test cases were only being created for debians (at least those are the only ones that appeared when I searched on all the [issues](https://github.com/osrf/ros2_test_cases/issues?page=5&q=is%3Aissue+Launch+testing)).

Comment (clalancette) [reply requirements/features-executable-from-links.yaml:30]: If you look at the closed ones as well (https://github.com/osrf/ros2_test_cases/issues?page=1&q=is%3Aissue+Launch+testing+is%3Aclosed), you'll see tests for Windows, from source, and from binary as well.

Comment (Voldivh) [reply requirements/features-executable-from-links.yaml:30]: Oh I see what you mean for Windows. Then wouldn't it makes sense to kinda leave it as it is? Perhaps add a little more instructions on what path to add on the command, e.g. `launch_test <path_to_share_folder>/launch_testing/examples/hello_world_launch_test.py` or something like that.

**Review by clalancette — APPROVED** [review]

**Pull request merged**